# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1290496683 18000
#      Tue Nov 23 02:18:03 2010 -0500
# Node ID 5abe6815aa5e6956652200b0cfafa509c9509598
# Parent  646063a22a35cddf4699f71c25118b8a98ef1818
install.txi: delete obsolete statement about needing a patch for makeinfo

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2010-11-23  John W. Eaton  <jwe@octave.org>
+
+	* interpreter/install.txi: Delete obsolete statement about
+	needing a patch for makeinfo.
+
 2010-11-18  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/contributors.in: Add Christos Dimitrakakis to the list.
 
 2010-11-18  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/contributors.in: Add Daisuke Takago to the list.
 
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -188,18 +188,17 @@ To compile Octave, you will need a recen
 also need a recent version of @code{g++} or other ANSI C++ compiler.  You
 will also need a Fortran 77 compiler or @code{f2c}.  If you use
 @code{f2c}, you will need a script like @code{fort77} that works like a
 normal Fortran compiler by combining @code{f2c} with your C compiler in
 a single script.
 
 If you plan to modify the parser you will also need GNU @code{bison} and
 @code{flex}.  If you modify the documentation, you will need GNU
-Texinfo, along with the patch for the @code{makeinfo} program that is
-distributed with Octave.
+Texinfo.
 
 GNU Make, @code{gcc}, and @code{libstdc++}, @code{gnuplot},
 @code{bison}, @code{flex}, and Texinfo are all available from many
 anonymous ftp archives.  The primary site is @url{ftp.gnu.org}, but it
 is often very busy.  A list of sites that mirror the software on
 @url{ftp.gnu.org} is available by anonymous ftp from
 @url{ftp://ftp.gnu.org/pub/gnu/GNUinfo/FTP}.
 
